src/GovUk.Education.ExploreEducationStatistics.Content.Model/Extensions/ReleaseAmendmentExtensions.cs

```diff
@@ -50,10 +59,20 @@ public static Release CreateAmendment(this Release release, DateTime createdDate
                 });
             }
 
-            amendment.ContentBlocks = amendment
-                .ContentBlocks
-                .Select(rcb => rcb.Clone(context))
-                .ToList();
+            amendment.KeyStatistics = amendment.KeyStatistics.Select(originalKeyStatistic =>
+            {
+                var amendmentKeyStatistic = originalKeyStatistic.Clone(context.NewRelease);
+
+                if (amendmentKeyStatistic.GetType() == typeof(KeyStatisticDataBlock))
```

Comment:
Again, you can use the type checking syntax here that will give you a new cast variable of type KeyStatisticDataBlock.